blueprints/hr_assistant/webex_bot_server.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+import os
+import logging
+import json
+from flask import Flask, request
+import requests
+from ciscosparkapi import CiscoSparkAPI
+
+from mindmeld.components import NaturalLanguageProcessor
+from mindmeld.components.dialogue import Conversation
+from mindmeld import configure_logs
+
+class WebexBotServer():
```

Comment:
Can we move this class to MindMeld so that you can do `from mindmeld import WebexBotServer`?